src/bin/add_filter.rs

```diff
@@ -75,17 +73,12 @@ async fn main() {
 
     let collection = db.collection::<mongodb::bson::Document>("filters");
 
-    // add a unique index on filter_id
-    match create_index(&collection, doc! {"filter_id": 1}, true).await {
-        Ok(_) => {}
-        Err(e) => {
-            error!("error creating index on filter_id: {}", e);
-        }
-    }
-
```

Comment:
Won't we want to index on this new id field?

Reply:
- If you are querying filters to have them deserialize/serialize from/to the struct directly, you just add a the following macro on top of your `id` field in the struct:

  ```
  #[serde(rename = "_id")]
  ```
  So when being read from/to mongo, the rust structs `id` is mapped to the document's `_id`. (I believe we might not be reading things directly as filter structs, which we should be.)
- if you are querying the document as is (not serializing/deserializing to/from a struct) then you can just use a projection, where you have:
  ```
  "id": "$_id"
  ```